Create README.md

imjorge opened the pull request.

lmars commented on README.md:
````diff
@@ -0,0 +1,110 @@
+$ go build
+```
+
+`flynn-discovery` requires a Postgres database to save cluster information. For this demo we will use a Docker container.
````
How about deploy it to a Flynn cluster which has Postgres already running :wink:

imjorge replied:
Added.

lmars commented:
@imjorge please add a [DCO sign off](https://flynn.io/docs/contributing#developer’s-certificate-of-origin) to the commits.

titanous commented:
We should drop the Docker example.

imjorge commented:
Removed Docker example and added DCO sign off. Although I think it would be useful for using `flynn-discovery` outside of Flynn which embodies the Flynn mantra of modularity where one could use components from it in isolation.

lmars commented:
@imjorge you need to add the DCO sign-off to each of the commits (you may as well squash them into one at the same time)

imjorge committed:
Create README.md

Signed-off-by: Jorge Ferreira <imjorge@gmail.com>

imjorge commented:
@lmars done.

lmars commented:
LGTM

lmars commented:
Thanks!
